internal/anclaHelper/anclaHelper.go

```diff
@@ -13,48 +13,25 @@ import (
 	"go.uber.org/zap"
 )
 
```

Comment:
moved to ancla: https://github.com/xmidt-org/ancla/pull/236/files#diff-c52daedaeda258db15ec11d57bbf299412a20328f6cd026980ef07f01960dd3a